Modules/Filtering/LabelMap/include/itkShapeLabelMapFilter.hxx

```diff
@@ -326,7 +326,7 @@ ShapeLabelMapFilter<TImage, TLabelImage>::ThreadedProcessLabelObject(LabelObject
   {
     principalMoments[i] = pm(i);
   }
-  MatrixType principalAxes = eigen.V.transpose();
+  MatrixType principalAxes(eigen.V.transpose());
```

Comment:
Note that strictly speaking, this pull request _might_ break _some_ user code, as it required the user to make the construction of an `itk::Matrix` from a ("non-fixed") `vnl_matrix` explicit. But the fix is usually straightforward, as you see here, using parentheses when declaring an `itk::Matrix` from a  `vnl_matrix`, instead of `=`.